### PR TITLE
fix!: replace config command with dev env

### PR DESCRIPTION
**Pull request by wtrocki**
Simple and clear implementation.
The bad news is that I could not get completion working

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -22,6 +22,15 @@ You can run the CLI commands files directly with `go run`. All commands and subc
 go run ./cmd/rhoas kafka create "test" --provider=aws --region=us-east-1
 ```
 
+### Development features
+
+To enable development and early access commands please set environment variable in your terminal:
+
+```bash
+RHOAS_DEV=true
```
Comment (craicoverflow) [inline CONTRIBUTING.md:30]:
```suggestion
export RHOAS_DEV=true
```

The help text will appear and completions will work only when RHOAS_DEV is exported.

File: pkg/profile/profile.go [hunk 2]
```diff
@@ -2,60 +2,29 @@
...
-}
+var DevPreviewEnv string = "RHOAS_DEV"
 
-// Annotation used in templates and tools like documentation generation
-func DevPreviewAnnotation() map[string]string {
-	return map[string]string{"channel": "preview"}
-}
+// ApplyDevPreviewLabel adds visual element displayed in help
+func ApplyDevPreviewLabel(cmd *cobra.Command) {
+	cmd.Short = "[preview] " + cmd.Short
```
Comment (craicoverflow) [inline pkg/profile/profile.go:14]: Maybe we could highlight this in a different colour?

Comment (wtrocki) [reply pkg/profile/profile.go]: I had that done. That will break ascidocs generation sadly and I could not figure out how to not include colours into docs

**Comment by wtrocki** [comment]
That PR  will lands into the dataplane branch. I would keep it open and merge the other one first (to not add more elements to review in the source branch.

**Comment by cben** [comment]
https://github.com/redhat-developer/app-services-cli/blob/main/docs/commands/rhoas_config.adoc file still remains, guess should be deleted.

**Comment by wtrocki** [comment]
@cben Amazing catch! Do you mind creating PR to remove it?

**Pull request merged**